### ISSUE TITLE
Can't start IPFS Desktop 0.10 on macOS

### ISSUE THREAD

**Issue by olizilla** [ipfs-shipyard/ipfs-desktop#1244]
I just got an update to IPFS Desktop and it failed to restart, it gave a code-signing error. I figured maybe it got corrupted... i was trying to restart my machine at them same time, so I downloaded the .dmg file from the github releases page, but after installing it, it fails to run. I can see this in my system logs

```
CODE SIGNING: process 856[IPFS Desktop]: rejecting invalid page at address 0x2c67d8dc2000 from offset 0x0 in file "<nil>" (cs_mtime:0.0 == mtime:0.0) (signed:0 validated:0 tainted:0 nx:0 wpmapped:1 dirty:0 depth:0)
```

**Comment by olizilla** [comment]
Downloading and installing 0.9.7 works.

**Comment by hacdias** [comment]
To mitigate the issue for now, I've set the 0.10.0 release as Pre-Release so no auto-update mechanism will upgrade to that by default.

This is an issue on Electron Builder (............) and the PR to fix it: https://github.com/electron-userland/electron-builder/pull/4414

Will leave a warning on the release page saying not to install that version.

**Comment by Lazerbeak12345** [comment]
Building from source on master was having this error as well recently. I'll build again later sometime within 4 days when I have the time.

**Comment by hacdias** [comment]
@Lazerbeak12345 the problem seems to be caused by electron-builder https://github.com/electron-userland/electron-builder/pull/4414

**Comment by Lazerbeak12345** [comment]
I'm aware. I noticed the link above. Just wanted to confirm. Sorry if that was unnecessary.

(PS: I noticed this like a week or two ago. I simply forgot to post the issue.)

**Comment by hacdias** [comment]
@Lazerbeak12345 no problem! I'm happy that someone is actually running this from source time to time!!

**Comment by hacdias** [comment]
Fixed on last #1265

**Comment by olizilla** [comment]
🎉
